network/policy/policy_windows.go

```diff
@@ -353,10 +353,12 @@ func GetHcnEndpointPolicies(policyType CNIPolicyType, policies []Policy, epInfoD
 		if policy.Type == policyType {
 			var err error
 			var endpointPolicy hcn.EndpointPolicy
+			var isOutbountNatPolicy bool
```

Comment:
You don't need this extra bool. You can just check if GetPolicyType(policy) is OutBoundNatPolicy

Reply:
Checked the code. having bool is good. Resolving the comment.